recipe-server/client/control_new/components/recipes/PreferenceExperimentFields.js

```diff
@@ -215,10 +221,10 @@ export class BooleanPreferenceField extends React.Component {
   render() {
     return (
       <Radio.Group {...this.props}>
-        <Radio.Button value={true}>
+        <Radio.Button value={true} id="pref-true">
```

Comment:
Won't this component get rendered multiple times on one page, making using an id here incorrect?